.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: csharp
+solution: Open-XML-SDK.sln
+install:
+  - nuget restore Open-XML-SDK.sln
+script:
+  - xbuild /p:Configuration=Release Open-XML-SDK.sln
+  # The following tests are currently not run due to a bug in Mono
+  # - mono ./packages/xunit.runner.console.2.0.0/tools/xunit.console.exe ./System.IO.Packaging.Tests/bin/Release/System.IO.Packaging.Tests.dll
+  # - mono ./packages/xunit.runner.console.2.0.0/tools/xunit.console.exe ./DocumentFormat.OpenXml.Tests/bin/Release/DocumentFormat.OpenXml.Tests.dll
```

Comment:
Due to the Mono bug, the tests are commented out for the time being. Once that bug is fixed, we can uncomment those lines.